src/utils/functions/fetchData.spec.js

```diff
@@ -0,0 +1,15 @@
+import fetchData from './fetchData';
+
+describe('Testing fetchData function', () => {
+  test('it should return an Array', async () => {
```

Comment:
This test fails if you are disconnected from internet. You should mock `fetch` API and assert based on your mock conditions. You could do:

```javascript
global.fetch = jest.fn().mockResolvedValue(DATA_YOU_WANT_RETURNED);
```

and for the failing cases:

```javascript
global.fetch = jest.fn().mockRejectedValue(YOUR_ERROR)
```

Remember to also do `global.fetch.mockRestore()` to retrieve the original behavior.